make/go/go.mk

```
@@ -84,14 +84,14 @@ godeps: deps
 .PHONY: gofmtmodtidy
 gofmtmodtidy:
 	@echo gofmt -s -w ALL_GO_FILES
-	@gofmt -s -w $(shell find . -name '*.go')
+	@find . -name '*.go' -print0 | xargs -0 gofmt -s -w
```

Isn't this equivalent to:

```suggestion
	@gofmt -s -w .
```

---

Yeah, that's a good point. I don't think there's any reason to use `find` here after all.